github/plugin.py

```diff
@@ -164,9 +166,11 @@ def github_issues(
             ),
             title=obj["title"],
             state=obj["state"],
-            body=obj.get("body", ""),
+            body=obj.get("body") or "",
```

Comment:
```suggestion
            body=obj.get("body", "")
```

?

Reply:
no, this doesn't work because if obj has `{"body": None}` then it gets `None`